internal/api/grpc/object/v2/converter.go

```diff
@@ -25,7 +25,7 @@ func DomainToDetailsPb(objectDetail *domain.ObjectDetails) *object.Details {
 func ToListDetails(response query.SearchResponse) *object.ListDetails {
 	details := &object.ListDetails{
 		TotalResult:       response.Count,
-		ProcessedSequence: response.Sequence,
+		ProcessedSequence: uint64(response.Position),
```

Comment:
this is not the same information... i see that we currently do not return the position and the sequence might not work / even be empty, but then we need to add a new field to the proto

Reply:
Ok, we can also move it around like that.